src/Box.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import './App.css';
+
+const Box = (props) => {
```

Comment:
Doesn't look like you're using this.

Reply:
Was going to be my product container, but I implemented separately. Removed from directory.